### DISCUSSION TITLE
Osu lazer

### DISCUSSION THREAD

**Discussion by Yousuck11** [ppy/osu#18812] [Q&A] [answered]
Is there a way to transfer bitmaps from osu lazer to osu?

**Comment by integerrr** [comment, marked as answer]
Use the "export package" option in the editor and then import it into osu-stable as normal
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/7279624/175180285-8df11607-9be2-4b66-8afb-fde896b88bae.png">
